docker/Dockerfile.server

```diff
@@ -1,12 +1,16 @@
 ARG BASE_SERVER_IMAGE=starwhaleai/base_server:latest
 FROM ${BASE_SERVER_IMAGE}
+ARG SW_SERVER_VERSION=server_version_not_set
+ARG SW_VERSION=cli_version_not_set
```

Comment:
remove `SW_VERSION`?  it is the same as `SW_VERSION`.

Reply:
the version of controller will show like `0.3.0-rc.5-nightly-20220920-a63f51c7:0.1.0.dev15:https://github.com/anda-ren/starwhale/tree/show_version`

`0.3.0-rc.5-nightly-20220920-a63f51c7` means the version of controller
`0.1.0.dev15` means the version of swcli(pypi)

Reply:
done